vars/clientUtils.groovy

```diff
@@ -336,3 +343,18 @@ def runAnsible(String playbookFile, String playbookTags=null){
         """
 
 }
+
+def copySshKey(Map parameters = [:]){
+    def sshKey = parameters.get("sshKey", 'ssh_credentials_remediations_fifi')
+    def sshKeyName = parameters.get("sshKeyName", 'insights-qa.pem')
+    withCredentials([file(credentialsId: sshKey, variable: 'settings')]) {
+        sh "cp \$settings ~/.ssh/${sshKeyName}"
+    }
+}
+
+def downloadChromeDriver(){
```

Comment:
Why do you download it every run? Would it be more efficient to have it in the image already?

Reply:
I moved this to iqe-ansible repo which preconfigures/creates our Jslave Openstack images.